### PR TITLE
Add ride-mode recipe

### PR DESCRIPTION
**Pull request by deadblackclover**
### Brief summary of what the package does

This package implements a major-mode for editing smart contracts written in RIDE.

### Direct link to the package repository

https://github.com/deadblackclover/ride-mode

### Your association with the package

Creator and maintaner

### Relevant communications with the upstream package maintainer

None needed

### Checklist

- [x] The package is released under a [GPL-Compatible Free Software License](https://www.gnu.org/licenses/license-list.en.html#GPLCompatibleLicenses)
- [x] I've read [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)
- [x] I've used the latest version of [package-lint](https://github.com/purcell/package-lint) to check for packaging issues, and addressed its feedback
- [x] My elisp byte-compiles cleanly
- [x] I've used `M-x checkdoc` to check the package's documentation strings
- [x] I've built and installed the package using the instructions in [CONTRIBUTING.org](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.org)

### CODE REVIEW

File: recipes/ride-mode [hunk 1]
```diff
@@ -0,0 +1 @@
+(twtxt :fetcher github :repo "deadblackclover/ride-mode")
```
Comment (riscy) [inline recipes/ride-mode:1]:
```suggestion
(ride-mode :fetcher github :repo "deadblackclover/ride-mode")
```

**Review by riscy — COMMENTED** [review]
Thanks for this - one fix with the recipe.

I also notice you're writing `RIDE` but it seems like -- if I have the URL right, at least -- their "branding" is to just write Ride in titlecase, e.g. https://github.com/wavesplatform/ride-introduction

File: recipes/ride-mode (hunk 1 above)
Comment (deadblackclover) [reply recipes/ride-mode]: Sorry, I was copying from my other package. Thx for fixing it!

**Comment by deadblackclover** [comment]
> Thanks for this - one fix with the recipe.
> 
> I also notice you're writing `RIDE` but it seems like -- if I have the URL right, at least -- their "branding" is to just write Ride in titlecase, e.g. https://github.com/wavesplatform/ride-introduction

Yeah, that's right.

**Comment by riscy** [comment]
> Yeah, that's right.

I presume you would want to align with that (maybe I should write that as a question?) but that's not a blocker on my end.  Merging - and thanks.

**Pull request merged**